src/main/kotlin/org/rust/lang/core/psi/RsFile.kt

```diff
@@ -65,6 +68,7 @@ class RsFile(
 
     override val isCrateRoot: Boolean
         get() {
+            if (originalFile.isDoctestInjection) return true
```

Comment:
Let's add a comment why we need this - looks like it can be not obvious in future reading